mne/externals/h5io/_h5io.py

```diff
@@ -84,6 +84,8 @@ def write_hdf5(fname, data, overwrite=False, compression=4,
         keys in data. This does not apply to the top level name (title).
         If 'error', '/' is not allowed in any lower-level keys.
     """
+    from ...utils import _validate_type
+
```

Comment:
do not change things in `mne/externals` we try to keep these as verbatim copies of the upstream versions